2017-11-04 14:19:35 database: wrong user or password; user: "ADMIN"
org.h2.message.DbException: Wrong user name or password [28000-195]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:104)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.test(WebApp.java:885)
	at org.h2.server.web.WebApp.process(WebApp.java:220)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 19 more
2017-11-04 15:30:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into TBL_PASSCODE  values ('admin', 'test123') [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-11-04 15:30:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into TBL_PASSCODE  values ('admin', 'test123') [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-11-04 15:31:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
insert into TBL_PASSCODE  values ("admin", "test123") [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-11-04 15:32:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO  TBL_PASSCODE   VALUES ("admin", "test123") [21002-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:265)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-11-04 15:32:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO  TBL_PASSCODE   VALUES (,[*]""admin"", ""test123"")"; expected "), DEFAULT, NOT, EXISTS, INTERSECTS, SELECT, FROM, WITH"; SQL statement:
INSERT INTO  TBL_PASSCODE   VALUES (,"admin", "test123") [42001-195]
2017-11-04 15:32:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "admin" not found; SQL statement:
INSERT INTO  TBL_PASSCODE   VALUES (0,"admin", "test123") [42122-195]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:262)
	at org.h2.engine.Session.prepareLocal(Session.java:573)
	at org.h2.engine.Session.prepareCommand(Session.java:514)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:165)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
